apps/allocations/app/components/App/App.js

```diff
@@ -38,7 +38,16 @@ const App = () => {
     tokenAddress,
   }) => {
     const emptyIntArray = new Array(addresses.length).fill(0)
-    console.log('budgetId: ', budgetId)
+    console.log('budgetId: ', budgetId, ' balance: ', balance)
+    console.table({
+      addresses,
+      description,
+      budgetId,
+      recurring,
+      period,
+      balance,
+      tokenAddress,
+    })
```

Comment:
Should we leave this in? Should we add `eslint-disable-line` statements, and a comment explaining why we want this in shipped code for now?